From 6d875bc78fbd10ff70a51d8c1a937711d0b32f25 Mon Sep 17 00:00:00 2001
From: Huw Davies <huw@codeweavers.com>
Date: Wed, 10 Jul 2019 09:50:17 +0100
Subject: [PATCH] gdi32: Use the correct type for the final parameter of
 FSOpenResourceFile().

This is SInt16 for 32-bit, but int for 64-bit.

Signed-off-by: Huw Davies <huw@codeweavers.com>
---
 dlls/gdi32/freetype.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dlls/gdi32/freetype.c b/dlls/gdi32/freetype.c
index 1162ea790d..e3cff25b76 100644
--- a/dlls/gdi32/freetype.c
+++ b/dlls/gdi32/freetype.c
@@ -818,7 +818,7 @@ static char *find_cache_dir(void)
 static char **expand_mac_font(const char *path)
 {
     FSRef ref;
-    SInt16 res_ref;
+    ResFileRefNum res_ref;
     OSStatus s;
     unsigned int idx;
     const char *out_dir;
-- 
2.17.1

